examples/react-todos/.meteor/packages

```diff
@@ -13,5 +14,6 @@ reactive-var
 meteorhacks:npm
 cosmos:browserify
 
+static-html
```

Comment:
Oh, does the .ng.html thing not do anything special with the Angular templates? I thought maybe you would want to have that act on the .html files directly.